addOns/authhelper/src/main/resources/org/zaproxy/addon/authhelper/resources/Messages.properties

```diff
@@ -38,3 +38,9 @@ The following tokens can be used in the values:\n\
 * {%script:glob_var%}	global script variable\n\
 * {%header:env_var%}	authentication response header\n\
 * {%url:key%}		authentication URL param
+
+authhelper.verification-detect.name = Verification Request Identified
+authhelper.verification-detect.desc = The given request has been identified as a good candidate for authentication verification. \
+TODO ... The 'Other Info' field contains a set of header tokens that can be used in the Header Based Session Management Method. \
```

Comment:
Do it?

Reply:
Ooops - had noticed this and done it locally, will push soon..